src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -71,6 +71,11 @@ public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionD
     }
 
     private void validateFullUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        // Method to be added to as Update journey developed
+        validateFullCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
+
+        ownersAndOfficersDataBlockValidator.validateOwnersAndOfficers(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
There is another validateOwnersAndOfficers call in the partial validation that should be enabled.